mmf/models/m4c.py

```diff
@@ -44,6 +45,19 @@ def build(self):
         self._build_mmt()
         self._build_output()
 
+    def _build_encoder_config(self):
+        return OmegaConf.create(
+            {
+                "type": "finetune_faster_rcnn_fpn_fc7",
+                "params": {
+                    "in_dim": 2048,
+                    "weights_file": "models/detectron.defaults/fc7_w.pkl",
+                    "bias_file": "models/detectron.defaults/fc7_b.pkl",
+                    "model_data_dir": self.config.model_data_dir,
```

Comment:
While you are here, maybe we can move these(`in_dim`, `weights_file`, `bias_file`) values to config?

Reply:
I didn't wanted to add this so as not to break older configs. That's why I skipped it for m4c.

Reply:
If there aren't a lot of m4c configs maybe it's better to move these there now, otherwise might cause problems later on if someone wants to change these values.

Reply:
Ok, let me handle this separately and create a task for it.